docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -39,6 +39,12 @@ of the project and those will go untested.
 To gain the ability to execute all aspects of the TinkerPop build system, other environmental configurations must be
 established. Those prerequisites are defined in the following subsections.
 
+As of TinkerPop 3.5.5, environment configuration for Gremlin Language Variants can be optional, as Docker becomes a system
```

Comment:
These first two sentences seem to be trying to explain the same thing.

Reply:
Deleted second one